### PR TITLE
ARM01 ci-operator dep buildConfig

### PR DESCRIPTION
**Pull request by deepsm007**
Adding buildconfig for ci-operator and its dependencies

/cc @droslean @danilo-gemoli

### CODE REVIEW

File: clusters/build-clusters/common/ci-operator/ci-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+kind: List
```
[the remark this answers is not in the document]
Comment (deepsm007) [reply clusters/build-clusters/common/ci-operator/ci-operator.yaml]: fixed and moved to arm01

**Review by droslean — APPROVED** [review]

**Pull request merged**